README.md

```diff
@@ -509,9 +509,39 @@
     [encapsulation](https://en.wikipedia.org/wiki/Encapsulation_(computer_programming)).
 
 * **What is the difference between a constructor and a method?** 
-    - The name of the constructor is same as that of the class name, whereas the name of the method can be anything.
-    - There is no return type of a constructor.
-    - When you make an object of a class, then the constructor of that class will be called automatically. But for methods, we need to call it explicitely.
```

Comment:
We can keep these lines.

Reply:
Yeah sure but don't I've already covered those points intuitively with example, should I now consider adding these again?

Reply:
Keep the above 3 points as it is and if the same is present in the latter half of the text, then you can remove that...